File: src/Cadastro.MVC/Models/Response/FuncionarioResponse.cs
using System;
using System.Collections.Generic;

namespace Cadastro.MVC.Models.Response
{
    public class FuncionarioResponse
    {
        public string UserId { get; set; }
        public string Matricula { get; set; }
        public string Cargo { get; set; }
        public string Nome { get; set; }
        public string SobreNome { get; set; }
        public string Email { get; set; }
        public DateTime? DataNascimento { get; set; }
        public List<TelefoneResponse> Telefones { get; set; } = new List<TelefoneResponse>();
        public EnderecoResponse EnderecoComercial { get; set; }
        public EnderecoResponse EnderecoResidencial { get; set; }
        public bool Ativo { get; set; }
    }
}

File: src/cadastro.angular/src/app/models/response/funcionarioResponse.ts
import { EnderecoResponse } from './enderecoResponse';
import { TelefoneResponse } from './telefoneResponse';

export interface FuncionarioResponse {
    userId: string;
    matricula: string;
    cargo: string;
    nome: string;
    sobreNome: string;
    email: string;
    dataNascimento: Date | null;
    telefones: TelefoneResponse[];
    enderecoComercial: EnderecoResponse;
    enderecoResidencial: EnderecoResponse;
    ativo: boolean;
}

